### PR TITLE
Blog - Grafana & Prometheus Kubernetes Cluster Monitoring

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks

### CODE REVIEW

**Comment by msali123** [comment]
@adamgordonbell 
image

Could you please look into this issue?

**Comment by jalletto** [comment]
@msali123 I want to reach out to the author and see if they can get a better image for Diagram showing the servers running the Namespace. Also, for all the image tags, let's delete the "Diagram showing" part. I left a couple comments with examples but please change for all.

**Comment by adamgordonbell** [comment]
@msali123  You can always disable a rule for a block of text Like this:
```
<!-- markdownlint-disable MD029 -->
4. If there is no database when PostgreSQL starts in a container, a default database will be created and it will not accept incoming connections during that time. This may cause issues with automation tools which may try to access the database as soon as the container starts. To mitigate this, you need to ensure that the database is accepting connections before trying to connect to it. If you're using [Docker Compose](https://docs.docker.com/compose/), you can use the [healthcheck](https://docs.docker.com/compose/compose-file/compose-file-v2/#healthcheck) feature:
<!-- markdownlint-enable MD029 -->

```
The number of the rule must be stated, MD034 in your case.

File: .github/styles/HouseStyle/tech-terms/k8s.txt [hunk 1]
```diff
@@ -12,6 +12,8 @@ Tekton
 tkn
 homebrew
 Statefulsets
+prometheus
```
Comment (adamgordonbell) [inline .github/styles/HouseStyle/tech-terms/k8s.txt:15]: Before you add words, make sure they are being used correctly @msali123 

It seems like this should be Prometheus, not lowercase:
https://prometheus.io/

File: .github/styles/HouseStyle/tech-terms/nontech.txt [hunk 2]
```diff
@@ -15,4 +15,6 @@ Residental
 rolodex
 versionable
 papercut
-YubiKey
\ No newline at end of file
+YubiKey
+Grap
```
Comment (adamgordonbell) [inline .github/styles/HouseStyle/tech-terms/nontech.txt:19]: I think this should be graph, and is a spelling mistake in the source document.

File: .github/styles/HouseStyle/tech-terms/k8s.txt (hunk 1 above)
Comment (msali123) [reply .github/styles/HouseStyle/tech-terms/k8s.txt:15]: Need to keep it for the links

**Comment by msali123** [comment]
> @msali123 I want to reach out to the author and see if they can get a better image for Diagram showing the servers running the Namespace. Also, for all the image tags, let's delete the "Diagram showing" part. I left a couple comments with examples but please change for all.

Okay I'll wait @jalletto

**Comment by msali123** [comment]
image
@jalletto error 😬

**Comment by msali123** [comment]
@jalletto could you please help with the issue.
Thanks

**Comment by jalletto** [comment]
@msali123 the author has put an updated image in the google doc so you should be able to grab it and replace it. As for the build failures, that's being worked on. If the build fails again after you push, ping me and I'll rerun it which should fix the issue.

**Comment by adamgordonbell** [comment]
@msali123  it looks like the author file is broken, that is the error it is producing.

**Comment by jalletto** [comment]
Ok looked into this ``` /usr/local/bundle/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': 
Thu, 25 Aug 2022 15:10:05 GMT         ./blog+build | /usr/local/bundle/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/site/_data/authors.yml): could not find expected ':' while scanning a simple key at line 250 column 1 (Psych::SyntaxError)```. There is a problem with the author page not parsing correctly.

**Comment by msali123** [comment]
> @msali123 I want to reach out to the author and see if they can get a better image for Diagram showing the servers running the Namespace. Also, for all the image tags, let's delete the "Diagram showing" part. I left a couple comments with examples but please change for all.

@jalletto I have this below link
But cant find the updated image
https://docs.google.com/document/d/15vFRBsp87vSKZBhihAkG0Asezpd3SMwy84adAXKrBhQ/edit

**Comment by msali123** [comment]
https://imgur.com/a/Of2tWDG
@jalletto is this the one you need?

File: .github/styles/HouseStyle/tech-terms/nontech.txt [hunk 3]
```diff
@@ -16,4 +16,8 @@ rolodex
 versionable
 papercut
 YubiKey
-skillsets
\ No newline at end of file
+<<<<<<< HEAD
+preconfigured
+=======
+skillsets
+>>>>>>> f06ad7e0cc36f828d8a05d373637118d12e24f5b
```
Comment (adamgordonbell) [inline .github/styles/HouseStyle/tech-terms/nontech.txt:23]: not valid yaml

**Comment by msali123** [comment]
@adamgordonbell is it now okay?

**Review by jalletto — APPROVED** [review]

**Pull request merged**